downstream/modules/devtools/proc-devtools-install-rpm.adoc

```diff
@@ -1,12 +1,11 @@
-[id="devtools-install_context"]
+[id="devtools-install_{context}"]
 
-= Installing {ToolsName} from an RPM package
+= Installing {ToolsName} from a package on RHEL
 
 [role="_abstract"]
 
 {ToolsName} is bundled in the {PlatformNameShort} RPM (Red Hat Package Manager) package.
-// As an {PlatformNameShort} administrator, you can install {ToolsName} when you are installing {PlatformNameShort}.
-Refer to the {PlatformNameShort} guide for more information on installing {PlatformNameShort}.
+Refer to the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}red_hat_ansible_automation_platform_installation_guide[_{PlatformName} Installation guide_] documentation for information on installing {PlatformNameShort}.
 
 .Prerequisites
 * You have installed RHEL
```

Comment:
In the preview there is two bullets here, but not in GH? The other being: "You have registered your system with Red Hat Subscription Manager."

The second uses a full stop while this one doesn't, if there actually is a second bullet and I'm not crazy, I'd suggest using full stops for both or neither, just keep it parallel.

Reply:
Weird. Both bullets are in the file. 
```
* You have installed RHEL.
* You have registered your system with Red Hat Subscription Manager.
```
Added the full stop.